src/java.base/share/classes/java/math/BigInteger.java

```diff
@@ -1296,10 +1296,10 @@ private static BigInteger valueOf(int[] val) {
     private static volatile BigInteger[][] powerCache;
 
     /** The cache of logarithms of radices for base conversion. */
-    private static final double[] logCache;
+    static final double[] logCache;
```

Comment:
I'd prefer this to remain `private`.

It's not a big deal to have `private static final double LOG_5_OF_2 = Math.log(2) / Math.log(5)` in `BigDecimal`.